.golangci.yml

```diff
@@ -32,10 +33,15 @@ linters:
 
 linters-settings:
   gocyclo:
-    min-complexity: 20 # default is 30.
+    # default is 30.
+    min-complexity: 20
+
+  goimports:
+    local-prefixes:
+      "github.com/mendersoftware/deviceconnect"
```

Comment:
all very cool, but why it is present here, while absent here: https://github.com/mendersoftware/devicemonitor/pull/69 ?

Reply:
@merlin-northern It is [there](https://github.com/mendersoftware/devicemonitor/pull/69/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9L40). It was there already.

Reply:
oh I see -- I checked but I could not find it. sorry for the noise.